src/eredis_client.erl

```diff
@@ -47,7 +47,7 @@
                 socket_options  :: list(),
                 tls_options     :: list(),
 
-                transport       :: tcp | tls | undefined,
+                transport       :: gen_tcp | ssl | undefined,
```

Comment:
Can this ever be undefined?

Reply:
ah, missed this, will put in next change